### ISSUE TITLE
svg-baker-runtime has wrong main file

### ISSUE THREAD

**Issue by vagusX** [kisenka/svg-baker#24]
```json
  "name": "svg-baker-runtime",
  "version": "1.3.4",
  "description": "",
  "license": "MIT",
  "repository": "https://github.com/kisenka/svg-baker/packages/svg-baker-runtime",
  "main": "browser-sprite.js",
  "module": "src/browser-sprite.js",
```

It seems that it didnot bundle out any assets?

**Comment by kisenka** [comment]
@vagusX yep, it was fixed in 1.3.5

**Comment by vagusX** [comment]
cool 👍
